apps/site/src/lib/components/forms/MultipleSelect.tsx

```diff
@@ -54,28 +58,27 @@ export default function RadioSelect({
 	return (
 		<div className={containerClass}>
 			<p className="m-0 text-lg mb-4">
-				{labelText} <span className="text-[#FF2222]">*</span>
+				{labelText} {isRequired && <RequiredAsterisk />}
 			</p>
 			<div
 				className={`w-10/12 flex ${
 					horizontal ? "flex-wrap gap-10" : "flex-col gap-2"
 				}`}
 			>
 				{values.map((item, i) => {
-					const keyandId = `${IdentifierId}-${i}`;
+					const inputId = `${name}-${i}`;
 					if (item.value === "other") {
 						return (
-							<div key={keyandId} className="flex gap-2">
+							<div key={i} className="flex gap-2">
```

Comment:
Thought: using `item.value` as the key should be sufficient and will also provide a warning if we mistakenly provided different options with the same value.